test_othermlpackages/ReadMe.md

```diff
@@ -0,0 +1,18 @@
+# Tests for other Machine Learning Packages
+
+The main test suite for Fairlearn uses estimators from SciKit-Learn.
+However, we wish to ensure that Fairlearn works with other machine learning packages, provided they implement the SciKit-Learn Estimator interface (with `sample_weight=` on the `fit()` method).
+This directory contains tests which demonstrate Fairlearn working with various other packages.
+
+We have made this a separate directory because machine learning packages tend to be heavyweight, and difference packages may have conflicting dependencies.
```

Comment:
typo "different" instead of "difference"

Reply:
Thanks